llarp/util/easter_eggs.cpp

```diff
@@ -1,39 +0,0 @@
-#include "lokinet_init.h"
-
-#if defined(_WIN32)
-#include <windows.h>
-#include <winuser.h>
-#include <stdio.h>
-
-#define WineKiller \
-  DieInCaseSomehowThisGetsRunInWineButLikeWTFThatShouldNotHappenButJustInCaseHandleItWithAPopupOrSomeShit
-
-struct WineKiller
-{
-  WineKiller()
-  {
-    if (auto hntdll = GetModuleHandle("ntdll.dll"))
-    {
-      if (GetProcAddress(hntdll, "wine_get_version"))
-      {
-        static const char* text =
-            "dont run lokinet in wine like wtf man we support linux and pretty "
-            "much every flavour of BSD, and even some flavours of unix system "
-            "5.x.\nThis Program Will now crash lmao.";
-        static const char* title = "srsly fam wtf";
-        MessageBoxA(NULL, text, title, MB_ICONHAND);
-        abort();
-      }
-    }
-  }
-};
-
-// i heckin love static initalization
-WineKiller lmao{};
-#endif
```

Comment:
The above should come back, but should move into `/daemon/lokinet.cpp` (so that it only fires on full lokinet, but not embedded lokinet).

Reply:
(It doesn't need to be in a struct; it could easily just be in the `main()` function)